common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -580,6 +581,17 @@ def _open_editor(self):
         self.course_wiki_page.open_editor()
         self.course_wiki_edit_page.wait_for_page()
 
+    def _check_for_accessibility_errors(self, page, custom_rules=None):
+        """ Run accessibility check with custom rules, if provided """
+        if custom_rules is None:
+            custom_rules = {
+                'ignore': ['section']  # AC-714, bug in the custom rule
```

Comment:
See AC-714 for details. The code has no section/heading issues now as far as I am aware (after the fixes in this PR).